templates/web/base/admin/index.html

```diff
@@ -64,5 +65,8 @@
         <tr><td>[% category.key %]</td><td>[% category.value.total %]</td><td>[% category.value.fixed / category.value.total * 100 | format('%.1f') %]%</td></tr>
     [% '</table>' IF loop.last %]
 [% END %]
+[% ELSE %]
+<p><a href="?show_categories=1">[% loc('Category fix rate for problems > 4 weeks old') %]</a></p>
```

Comment:
Might be nice to include an anchor in the URL hash so the page scrolls directly to the table when the link is followed?